Automattic/jetpack
# Social: Fix attached media disappearing on post publish.

Fixes 1204116019981263-as-1203876111187914

## Proposed changes:

* When the post gets published, all the connections get toggled off. Due to this, the attached media gets a validation error, due to which the attached media gets removed from the local store. This PR fixes that. 

### Other information:


## Jetpack product discussion


1204116019981263-as-1203876111187914
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






* Without this branch, try publishing a post with an image attached and notice that on publish, when you toggle the social media connection, the image doesn't show up. 
*On this branch, try publishing a post with an image attached and notice that on publish, when you toggle the social media connection, the image will show up. 

Run the test with `Run the tests with:
npm run test -- src/hooks/use-media-restrictions/test/index.test.js in projects/js-packages/publicize-components`
